### PR TITLE
Fixes #34 replace call_user_func* with self invoked function

### LINKED ISSUE #34: replace call_user_func* with self invoked function
as dev-master already use php ^7.1

---

Originally posted by @samsonasik at https://github.com/zendframework/zend-mail/pull/200

### PR DESCRIPTION
**Pull request by samsonasik**
Signed-off-by: Abdul Malik Ikhsan <samsonasik@gmail.com>

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

As requirement already upped to php ^7.1, I recreate PR https://github.com/zendframework/zend-mail/pull/200 Fixes https://github.com/laminas/laminas-mail/issues/34

### CODE REVIEW

File: src/Transport/File.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ public function setOptions(FileOptions $options)
     public function send(Message $message)
     {
         $options  = $this->options;
-        $filename = call_user_func($options->getCallback(), $this);
+        $filename = $options->getCallback()($this);
```
Comment (Ocramius) [inline src/Transport/File.php:73]: This kind of change should probably go to `develop`

Comment (samsonasik) [reply src/Transport/File.php:73]: we already have php 7.1 requirement, why need to go to develop?

Comment (Ocramius) [reply src/Transport/File.php:73]: This is not a bugfix: only bugfixes for stable releases :)

Comment (samsonasik) [reply src/Transport/File.php:73]: ok, changed to develop 👍

**Comment by Ocramius** [comment]
Thanks @samsonasik!

**Comment by weierophinney** [comment]
I've merged this into the new 2.13.x branch at this time; see 9be32067

**Pull request merged**